app/templates/request/_edit_requester_info.html

```diff
@@ -16,49 +17,57 @@
                         aria-hidden="true">&times;</span></button>
                 <h4 class="modal-title">Requester Information</h4>
             </div>
-            <form id="user-info" action="/request/edit_requester_info/{{ request.id }}" method="POST">
+            <form id="user-info">
                 {{ edit_requester_form.csrf_token }}
 
                 <div class="modal-body">
+                    {% set public_requester = request.requester.is_public %}
```

Comment:
I'd say make it `is_public_requester`. It is a boolean value, so the variable name should reflect that as well. `public_requester` makes me think that it is a requester object.